lib/Modules/HyperVCIHooks/HyperVCIHooks.psm1

```diff
@@ -1327,15 +1331,20 @@ function Start-InstallHook {
         python "$PYTHON_DIR\Scripts\pywin32_postinstall.py" -install
     } -ErrorMessage "Failed to run pywin32_postinstall.py"
 
-    Write-JujuLog "Running Git Prep"
     $zuulUrl = Get-JujuCharmConfig -Scope 'zuul-url'
     $zuulRef = Get-JujuCharmConfig -Scope 'zuul-ref'
     $zuulChange = Get-JujuCharmConfig -Scope 'zuul-change'
     $zuulProject = Get-JujuCharmConfig -Scope 'zuul-project'
-    $gerritSite = $zuulUrl.Trim('/p')
-    Start-GerritGitPrep -ZuulUrl $zuulUrl -GerritSite $gerritSite -ZuulRef $zuulRef `
-                        -ZuulChange $zuulChange -ZuulProject $zuulProject
-
+    if (($zuulProject -eq "openstack/cinder") -or (!$zuulProject)) {
```

Comment:
Wrong. If project is neutron or nova, run gerrit-git-prep for it. Also, in case it's os-win or something else what happens? I'd consider here an "include" list.